### ISSUE TITLE
Adding Search Filters to Operations table

### ISSUE THREAD

**Issue by hannavovk** [bcgov/cas-registration#1282]
#### Description:

As a CAS internal user, I want to be able to search operation BORO ID applications so I don't have to scroll through a long list to find the application I am looking for.
 
The wireframes of Registration pt. 1 in Figma included search boxes, that would work as filters, under column titles in the dashboards (screenshot 1). 
 
Could we please get these added in the dashboard (screenshot 2)? They would be very useful as applications are piling up.

![Image](https://github.com/bcgov/cas-registration/assets/87211104/81a10b25-c5fb-4654-bafb-328e2cfbd3ec)

![Image](https://github.com/bcgov/cas-registration/assets/87211104/54335373-4869-4489-858b-22f55ef9ce6e)

 
 
#### Acceptance Criteria:

Given
When
Then

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by
- Blocking

**Comment by hannavovk** [comment]
@marcellmueller investigating how to make a quick fix and will create a separate ticket for the search box design after

**Comment by shon-button** [comment]
@marcellmueller 

[Next.js 14: The Power of the URL...](https://blog.gogrow.dev/next-js-14-the-power-of-url-in-state-management-7d38a4e34ca8)

```
One of the most compelling aspects of using the URL as a state manager is the seamless ability to share curated experiences effortlessly...the experience becomes dynamic and easily shareable.

Take, for instance, the URL: https://www.examplestore.com/product-page?color=pink&size=XL. This link encapsulates a specific state within the application – in this case, showcasing products tailored to the user's preference for pink color and XL size. The magic happens when this link is shared.
```

**Comment by hannavovk** [comment]
@marcellmueller For some reason I'm not seeing the search boxes in Dev (and Test is not showing me any operations at all, which is probably something different - I guess I have to go submit some applications first :) )


![Image](https://github.com/bcgov/cas-registration/assets/87211104/ab80729b-d917-4187-a40b-cafdac49d623)
